quickwit/quickwit-control-plane/src/lib.rs

```diff
@@ -45,21 +44,24 @@ pub struct IndexerNodeInfo {
 
 pub type IndexerPool = Pool<String, IndexerNodeInfo>;
 
-/// Starts the Control Plane.
-pub async fn start_indexing_scheduler(
-    cluster_id: String,
-    self_node_id: String,
-    universe: &Universe,
-    indexer_pool: IndexerPool,
-    metastore: Arc<dyn Metastore>,
-) -> anyhow::Result<Mailbox<IndexingScheduler>> {
-    let scheduler = IndexingScheduler::new(cluster_id, self_node_id, metastore, indexer_pool);
-    let (scheduler_mailbox, _) = universe.spawn_builder().spawn(scheduler);
-    Ok(scheduler_mailbox)
-}
-
 #[derive(Debug, Clone)]
-pub struct ControlPlaneEventSubscriber(pub ControlPlaneServiceClient);
+pub struct ControlPlaneEventSubscriber(ControlPlaneServiceClient);
```

Comment:
I need more info on this. Reading the code I could not tell if this was sending notification to the ControlPlane or the other way around :-S

Reply:
Added a comment.